webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -9907,13 +9926,15 @@ var KM_ATTACHMENT_V2_SUPPORTED_MIME_TYPES = ["application","text","image"];
                             };
                         };
                         if (messageType === "APPLOZIC_01" || messageType === "MESSAGE_RECEIVED") {
-                            var messageFeed = mckMessageLayout.getMessageFeed(message);
-                            Kommunicate.KmEventHandler.onMessageReceived(message);
+                            // var messageFeed = mckMessageLayout.getMessageFeed(message);
+                            var messageCopy = JSON.parse(JSON.stringify(message));
```

Comment:
is this equivalent to mckMessageLayout.getMessageFeed(message)?
above function sets some values for file attachment URLs etc, please check if those are equivalent

Reply:
```
var messageFeed = mckMessageLayout.getMessageFeed(message);
```
This line was not getting used as the code below was commented. So that's why I commented the above line.